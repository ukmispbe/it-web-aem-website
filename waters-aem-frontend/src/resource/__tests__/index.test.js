import React from 'react';
import { shallow } from 'enzyme';

import Resource from '../index';
import props from './props';

describe('<Resource />', () => {
    let enzymeWrapper;
    beforeAll(() => {
        enzymeWrapper = shallow(<Resource {...props} />);
    });
    it('should render without throwing an error', () => {
        expect(enzymeWrapper).toMatchSnapshot();
    });
    it('should display Resources', () => {
        expect(enzymeWrapper.exists('.cmp-resource-section')).toEqual(true);
        expect(enzymeWrapper.exists('h4')).toEqual(true);
        expect(enzymeWrapper.find('h4').text()).toEqual(props.title);
        expect(enzymeWrapper.exists('.cmp-resource-section__list')).toEqual(true);
        expect(enzymeWrapper.exists('ul')).toEqual(true);
    });
});