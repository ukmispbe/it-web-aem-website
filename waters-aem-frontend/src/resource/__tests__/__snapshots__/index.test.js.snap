// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Resource /> should render without throwing an error 1`] = `ShallowWrapper {}`;
